CHANGES_NEXT_RELEASE

```diff
@@ -8,3 +8,4 @@
 - [cygnus-ngsi][hardening] Add raw bytes to NGSIEvent (#1345)
 - [cygnus-ngsi][hardening] Add cache to NGSIMySQLSink (#130)
 - [cygnus][hardening] Fix "FIWARE" name in license headers (#1369)
+- [cygnus-ngsi][bug] Differentiate among enterprise and individual accounts (#1382)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1393/commits/f2d7e43266253d141ea7e0f83ade64436c238e59